troupon/static/scss/partials/_resets.scss

```diff
@@ -49,8 +49,14 @@ ul, ol {
     position: absolute;
     z-index: z-index(dropdown);
 
-    li > a {
+    & > li {

```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Unnecessary parent selector (&)